### PR TITLE
Feature/list dropdown

### PR DESCRIPTION
**Pull request by janitastic**
### What was changed/implemented? (Why and How)
- Updated the MyList component to be a class for gottaGo and lovedIt list displays.
- Added create new list button

### Is it a feature/refactor/bugfix?
Refactor.

### Next Steps:
- Cypress testing.
- Write functionality for 'add new list' button.

### CODE REVIEW

File: src/Components/MyLists/MyLists.tsx [hunk 1]
```diff
@@ -8,46 +8,68 @@ interface Props {
...
-      rating={restaurant.rating}
-      image={restaurant.image_url}
-      location={restaurant.location.display_address}
-      phone={restaurant.phone}
-      url={restaurant.url}
-      />
-    )
-  })
+interface State {
+  cardsToDisplay: JSX.Element[], // need to double check how to type this
```
Comment (janitastic) [inline src/Components/MyLists/MyLists.tsx:12]: @hannahhch is `JSX.Element[]` an TS appropriate way to assign type? Or do we need to create an interface specifically for the cardToDisplay.

File: src/Components/ListCard/ListCard.scss [hunk 2]
```diff
@@ -1,7 +1,7 @@
 @use '../../styles/_variables' as v;
```
Comment (hannahhch) [inline src/Components/ListCard/ListCard.scss:1]: You got `@use` working, love it!

File: src/Components/MyLists/MyLists.scss [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+//desktop requirements
+@media screen and (min-width: 40em) {
+  .hidden-desktop {
+    display: none;
+  }
+}
+
+//mobile requirements
+@media screen and (max-width: 40em) {
```
Comment (hannahhch) [inline src/Components/MyLists/MyLists.scss:41]: I love the change you made to your menu for mobile! Just make sure its still as easy to get to as it is on desktop (right now a lot of scrolling has to happen first).

File: src/Components/MyLists/MyLists.tsx (hunk 1 above)
Comment (hannahhch) [reply src/Components/MyLists/MyLists.tsx:12]: This works fine! You'd only make an interface if there was something extra you wanted to require here.

**Review by hannahhch — COMMENTED** [review]
Looks great so far, nice work!

**Pull request merged**